packages/react/src/template.tsx

```diff
@@ -47,11 +47,31 @@ export type ActionDataState<T = any> = {
   $: 'action'
   name: T
 }
+export type ResponseDataState = {
+  $: 'response'
+  payload: JSONValue
+  statusCode: number
+  ok: boolean
+  actions: ActionDataState[]
+}
+export type HandlerDataState = {
+  $: 'handler'
```

Comment:
oh so this is used for the server state only? and it gets converted to an event before sent to the client?

Reply:
Removed this so ignore.